tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py

```diff
@@ -34,6 +34,12 @@ def test_browser_bookmark(self):
         bookmark.type_bookmark_title(self.bookmark_title)
         bookmark.tap_add_bookmark_to_home_screen_dialog_button()
 
+        # tap menu button to test to see if the add to homescreen button is removed
+        browser.switch_to_chrome()
+        browser.tap_menu_button()
+        self._add_to_home=bookmark.is_add_to_home_button_not_in_dialog()
+        self.assertFalse(self._add_to_home, 'Add to Homescreen is not in menu options')
```

Comment:
This message is only used [when the assertion fails](https://github.com/python/cpython/blob/2.7/Lib/unittest/case.py#L412). So it would be more clear to write: "Add to Homescreen should not be in menu options"